packages/lab/src/Wizard/Wizard.d.ts

```diff
@@ -0,0 +1,76 @@
+import * as React from "react";
+import { StandardProps, ModalProps } from "@material-ui/core";
+import { Breakpoint } from "@material-ui/core/styles/createBreakpoints";
+import { HvWizardActionsProps } from "./WizardActions";
+import { HvWizardTitleProps } from "./WizardTitle";
+
+export type HvWizardClassKey = "root";
+
+export type HvWizardProps = StandardProps<
+  React.HTMLAttributes<HTMLDivElement>,
+  HvWizardClassKey
+> & {
+  /**
+   * Class names to be applied.
+   */
+  className: string;
+  /**
+   * Current state of the Wizard.
+   */
+  open: boolean;
+  /**
+   * Function executed on close.
+   */
+  onClose: ModalProps["onClose"];
+  /**
+   * Function executed on submit.
+   */
+  onSubmit: any;
+  /**
+   * Title for the wizard.
+   */
+  title: string;
+  /**
+   * The content of the component.
+   */
+  children: React.ReactNode;
+  /**
+   * An object containing all the labels for the wizard.
+   */
+  labels: HvWizardActionsProps["labels"] & HvWizardTitleProps["labels"];
+  /**
+   * Shows the summary button.
+   */
+  hasSummary?: boolean;
+  /**
+   * Enables the skip button.
+   */
+  skippable?: boolean;
+  /**
+   * Forces minimum height to the component.
+   */
+  fixedHeight?: boolean;
+  /**
+   * Whether the loading animation is shown.
+   */
+  loading?: boolean;
+  /**
+   * Custom object to define type, size and width of the StepNavigation component
+   */
+  customStep?: {
```

Comment:
If you put Pick<HvStepNavigationProps, "type" | "size" | "width"> it will be always protected for future changes on StepNavigation component